debian/control

```diff
@@ -3,7 +3,7 @@ Section: admin
 Priority: optional
 Homepage: https://cloud-init.io/
 Maintainer: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
-Build-Depends: debhelper (>= 9.20160709),
+Build-Depends: debhelper-compat (= 10),
```

Comment:
I think this should be safe as we're packaging the same codebase to Bionic with compat level 9 (per `ubuntu/bionic branch`). However this requires checking what changes between different compat versions, see `man debhelper` or `man debhelper-compat-upgrade-checklist` if you're running Kinetic.

I had a look and the changeset looks safe (but please double check!), but in principle this is a nontrivial change.

Reply:
I checked the changes superficially, and I did not notice we were hitting a breaking change. `sbuild` runs correctly too. Therefore, I would say it is safe.